lbrynet/analytics/events.py

```diff
@@ -23,30 +23,38 @@ def __init__(self, context, lbry_id, session_id):
         self.session_id = session_id
 
     def heartbeat(self):
-        return {
-            'userId': 'lbry',
-            'event': 'Heartbeat',
-            'properties': {
-                'lbry_id': self.lbry_id,
-                'session_id': self.session_id
-            },
-            'context': self.context,
-            'timestamp': utils.now()
-        }
+        return self._event('Heartbeat')
 
     def download_started(self, name, stream_info=None):
+        properties = {
+            'name': name,
+            'stream_info': get_sd_hash(stream_info)
+        }
+        return self._event('Download Started', properties)
+
+    def metric_observed(self, metric_name, value):
```

Comment:
How do you feel about enumerating all the possible metric names. idk what the pythonic way of doing this is. In php, I used class constants. In ruby, this would probably be a symbol. The idea is that this is not a string, its a unique identifier that represents the same thing everywhere in the code. pycharm should be able to tell me every place where this thing appears.

In general, I hate using strings for anything other that input/output.